webvr_ar_extension.md

```diff
@@ -133,6 +133,76 @@ display.getPlanes().forEach(plane => {
 });
 ```
 
+### Anchors
+
+AR systems tyr to estimate with the highest accuracy possible the pose of the device
+in the real world. This estimation evolves over time as the system "learns" more about
+the real world and because of this learning process, the pose estimation may vary to
+be as accurate as it can get.
+
+An anchor is the concept of specifying to the underlying tracking system that a especific
+pose in the world space is important for the app and that if the system should track changes
+in the device pose estimation to correctly update the important pose at all times.
```

Comment:
I would add a paragraph about how the updating scheme works:
it's time based, it's a revision of what the phone used to think was correct, all anchors after the timestamp update.

I think the most confusing part of anchors is in the update scheme.

Reply:
I added a bit more info but I do not really think providing too much specifics on this matters too much. You create an anchor with a pose and the system will notify you when its updated because the tracking estimation has evolved. That's all I think a developer should understand/know.

Reply:
True; a lot of the complexity is abstracted away once over the web barrier. There was some misunderstanding I had though regarding the updates, but I think @mprafson cleared it up for me; if I recall what the issue is (and if it's not addressed in this description), I can add that update